client/python/armada_client/event.py

```diff
@@ -0,0 +1,102 @@
+import enum
+import typing
+from dataclasses import dataclass
+
+import google.protobuf.timestamp_pb2 as timestamp_pb2
+
+from armada_client.armada import event_pb2, queue_pb2
+from armada_client.k8s.io.apimachinery.pkg.api.resource import (
+    generated_pb2 as resource_pb2,
+)
+
+
+class EventType(enum.Enum):
```

Comment:
Can you add some unit tests for this class?

Reply:
I'd like to see accessing members of these events and aiming to test all the enums.